### ISSUE TITLE
Transcribe RTL script to LTR script

### ISSUE THREAD

**Issue by johnlockejrr** [mittagessen/kraken#669]
Is there an achievable way to transcribe a RTL script (Avestan) to a LTR script (Latin)? If so, how can be done? I don't see an option out of the box.

**Comment by mittagessen** [comment]
On 25/01/01 04:10AM, johnlockejrr wrote:
> Is there an achievable way to transcribe a RTL script (Avestan) to a
> LTR script (Latin)? If so, how can be done? I don't see an option out
> of the box.

You could teach a standard recognition model to output the LTR
transliteration for the input but that doesn't work particularly well in
my experience as the structure of the CTC loss requires the network to
'store' the recognition results internally to then emit them reversed
into the output.

The new recognizer doesn't have this issue as it isn't trained with CTC.
There will be a new base model for that one by next week.

**Comment by johnlockejrr** [comment]
Perfect! Looking forward!

**Comment by johnlockejrr** [comment]
Any update on this?

**Comment by mittagessen** [comment]
On 25/01/09 01:51PM, johnlockejrr wrote:
> Any update on this?

The model is up now. Instructions are in the
[party](https://github.com/mittagessen/party) repository.

**Comment by johnlockejrr** [comment]
For segmentation, is the new `blla` model on zenodo? Or the `Pretrained multilingual Party model`?

**Comment by johnlockejrr** [comment]
Can I try to finetune the model with a `NVIDIA GeForce RTX 4070 12Gb`? Worked fine for kraken models.

**Comment by mittagessen** [comment]
The new segmentation model isn't there yet because it didn't converge
for whatever reason so I need to debug that. 

The pretrained party model is the right one for the new recognizer but
it will also download it for you if you run the inference stuff without
an explicit model given. Right now you'll need ~20Gb of memory for
fine-tuning but I'll add some small changes later today that will reduce
memory consumption.

You can run inference with large batch sizes on that GPU though.

**Comment by niktayek** [comment]
Hi @johnlockejrr 
I have successfully trained an Avestan transliteration model that receives an Avestan manuscript page and returns the transliteration (Latin script). Do you want to get in touch?

**Comment by johnlockejrr** [comment]
Wow! Defenitely!
johnlockejrr [at] gmail.com
